lib/matplotlib/image.py

```diff
@@ -13,17 +13,14 @@
 import PIL.PngImagePlugin
 
 import matplotlib as mpl
-from matplotlib import _api
+# For clarity, names from _image are given explicitly in this module
+from matplotlib import _api, _image, cbook, cm
```

Comment:
This comment is a bit confusing now that you've joined up all the imports into one line.

Reply:
I agree. There are some remaining cases which I have not changed (as there are other PRs that touch those lines), so to save a test build, I can improve it in an upcoming PR once the other PRs are merged.

Reply:
Oh, but there is a conflict now, so I'll update this.